src/gnu/nrnran123.cpp

```diff
@@ -26,6 +26,13 @@ std::uint32_t nrnran123_get_globalindex() {
 nrnran123_State* nrnran123_newstream(std::uint32_t id1, std::uint32_t id2) {
     return nrnran123_newstream3(id1, id2, 0);
 }
+
+nrnran123_State* nrnran123_newstream0() {
+    extern int nrnmpi_myid;
+    static std::uint32_t id3{};
```

Comment:
`just a note`: `static` variables are not allowed on GPU / OpenACC but streams are always allocated on CPU